docs/user/algo_dqn.md

```diff
@@ -0,0 +1,95 @@
++-------------------+------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| **Code**          | `garage/torch/algos/dqn.py <https://github.com/rlworkgroup/garage/blob/master/src/garage/torch/algos/dqn.py>`_   | `garage/tf/algos/dqn.py <https://github.com/rlworkgroup/garage/blob/master/src/garage/tf/algos/dqn.py>`_     |
++-------------------+------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| **Examples**      | `examples <algo_dqn.html#examples>`_                                                                                                                                                                                            |
++-------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+Deep Q Networks, or simply DQN, is a staple off-policy method ontop of which many more recent algorithms were developed. It uses a learned Q function to obtain estimates for the values of each state, action pair (S,A), and selects the optimal value by simply taking the argmax of all Q values. The algorithm is most known for learning policies that perform well on a large suite of Atari games.
+
+## Supported DQN Variants
+
+As of now, both the Tensorflow and PyTorch implementations support Double DQN and Dueling DQN, as well as both combined. Double DQN can be used by simply passing a flag to the algorthim:
+
+```python
+algo = DQN(policy=policy,
+           double_q=True,
+           ...)
+```
+
+Dueling DQN can be used by passing in a Q function with the dueling architecture to algorithm. When using the Tensorflow branch, a dueling Q function can be constructed as follows:
+
+```python
+qf = DiscreteCNNQFunction(env_spec=env.spec,
+                          dueling=True,
+                          ...)
+```
+
+See the Pong example below for a full launcher file.
+
+In the Pytorch branch, the architecture belongs to its own class:
+
+```python
+qf = DiscreteDuelingCNNQFunction(env_spec=env.spec,
+                                 ...)
+```
+
+See the Atari launcher below for a full example.
+
+## Examples
+
+Both the Tensorflow and PyTorch branches of garage contain several DQN examples.
+
+### Pytorch
+
+The PyTorch branch provides an Atari launcher (see below) that allows you to easily run experiments from the command line like so:
+
+```bash
+# train on PongNoFrameskip-v4 for 10M steps
+python examples/torch/dqn_atari Pong --seed 22 --n_steps 10e6
```

Comment:
i don't think this line executes? also keep in mind that we should assume users are working from a pip distribution, not the development repo.

Reply:
Fixed, this should run on the latest release.